test/Microsoft.NET.Build.Tests/GivenThatWeWantToReferenceAProject.cs

```diff
@@ -41,15 +41,13 @@ public enum ReferenceBuildResult
         }
 
         [Theory]
-        [InlineData("netcoreapp1.1", true, "netstandard1.5", true, true, true)]
         [InlineData("netstandard1.2", true, "netstandard1.5", true, false, false)]
         [InlineData("netcoreapp1.1", true, "net45;netstandard1.5", true, true, true)]
         [InlineData("netcoreapp1.1", true, "net45;net46", true, false, false)]
         [InlineData("netcoreapp1.1;net461", true, "netstandard1.4", true, true, true)]
         [InlineData("netcoreapp1.1;net45", true, "netstandard1.4", true, false, false)]
         [InlineData("netcoreapp1.1;net46", true, "net45;netstandard1.6", true, true, true)]
         [InlineData("netcoreapp1.1;net45", true, "net46;netstandard1.6", true, false, false)]
-        [InlineData("v4.6.1", false, "netstandard1.4", true, true, true)]
```

Comment:
Why are two tests here removed? I see there's overlap between this and new tests but don't understand what makes these two and only these two redundant.